daemon/stop.go

```diff
@@ -54,8 +52,11 @@ func (daemon *Daemon) containerStop(container *containerpkg.Container, seconds i
 	stopSignal := container.StopSignal()
 
 	// 1. Send a stop signal
-	err := daemon.killPossiblyDeadProcess(container, stopSignal)
+	err := daemon.killPossiblyDeadProcess(ctx, container, stopSignal)
 	if err != nil {
+		if ctx.Err() != nil {
```

Comment:
No specific comment, just a note to other reviewers to verify if the logic in this code is good (I know we had a lot of logic issues in these parts of the code, so could use more eyes on this file/function)